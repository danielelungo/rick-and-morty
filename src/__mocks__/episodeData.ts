export const episodeData = [
  { date: "December 2, 2013", id: 1, name: "Pilot", episode: "S01E01" },
  { date: "December 9, 2013", id: 2, name: "Lawnmower Dog", episode: "S03E03" },
];
