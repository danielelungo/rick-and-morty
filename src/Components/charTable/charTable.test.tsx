import React from "react";
import renderer from "react-test-renderer";
import { episodeData } from "../../__mocks__/episodeData";
import CharTable from "./charTable";

describe("CharTable component", () => {
  test("renders CharTable Component", () => {
    const tree = renderer.create(<CharTable data={episodeData} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
