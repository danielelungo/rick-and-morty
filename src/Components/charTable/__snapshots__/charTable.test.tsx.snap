// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CharTable component renders CharTable Component 1`] = `
<div>
  <div
    className="sc-bczRLJ hBJfIz"
  >
    <div
      className="sc-gsnTZi GLudu"
    >
      Ep NÂ°
    </div>
    <div
      className="sc-dkzDqf iacFKi"
    >
      Name
    </div>
    <div
      className="sc-hKMtZM gerlvS"
    >
      Date
    </div>
  </div>
  <div
    className="sc-eCYdqJ kSjEDL"
  >
    <div
      className="sc-jSMfEi jrxkFZ"
    >
      1
    </div>
    <div
      className="sc-gKXOVf hRgWju"
    >
      Pilot
    </div>
    <div
      className="sc-iBkjds duicVq"
    >
      December 2, 2013
    </div>
  </div>
  <div
    className="sc-eCYdqJ kSjEDL"
  >
    <div
      className="sc-jSMfEi jrxkFZ"
    >
      2
    </div>
    <div
      className="sc-gKXOVf hRgWju"
    >
      Lawnmower Dog
    </div>
    <div
      className="sc-iBkjds duicVq"
    >
      December 9, 2013
    </div>
  </div>
</div>
`;
